gateway/assets/js/faas-ui.js

```diff
@@ -0,0 +1,23 @@
+angular
+  .module('faasUI', ['ngMaterial'])
+  .controller('faasCtrl', function($mdSidenav, $scope) {
+    $scope.avatar = function(func) {
+      return 'https://www.gravatar.com/avatar/' + md5(func.name) + '?d=identicon&f=y';
```

Comment:
I'm not keen on this change.. sorry. Please can we revert to the way @kenfdev had it? I don't want to have N XHR calls going out if we can avoid it.

Reply:
As in the store? No problem for me.